src/snapred/resources/application.yml

```diff
@@ -121,6 +123,9 @@ samples:
 
 cis_mode: false
 
+temp:
+  directory: /SNAPRed
```

Comment:
Will this mess up write permissions on analysis?

Reply:
I dont think so?  This can actually be removed since I moved to using the proper tempfile call